mimic_joint/proposal.md

```diff
@@ -161,14 +161,23 @@ and one `gearbox` joint.
         </joint>
 ~~~
 
-The `gearbox` joint could be equivalently and more concisely expressed as
-the following `mimic` joint:
+The `gearbox` joint could replaced equivalently by adding the ``<mimic>``
+tag to joint axes :
 
 ~~~
-        <joint name="mimic_demo" type="mimic">
-            <parent>gearbox_input_joint</parent>
-            <child>gearbox_output_joint</child>
-            <gearbox_ratio>5</gearbox_ratio>
+        <!-- Gearbox links revolute joints, so create a couple revolute joints -->
+        <joint name="gearbox_input_joint" type="revolute">
+            <parent>gearbox_base</parent>
+            <child>gearbox_input</child>
+            <axis>
+                <xyz>1 0 0</xyz>
+                <mimic joint="gearbox_output_joint" multiplier="5"/>
```

Comment:
the `//mimic` tag can be applied to either joint, but I think it makes more sense semantically to specify the `//mimic` tag in `gearbox_output_joint`

Reply:
Done https://github.com/gazebosim/sdf_tutorials/pull/80/commits/e708aec542099e2c10dc9d562148be4299a466cf